src/migmose/__main__.py

```diff
@@ -45,6 +45,7 @@ def main(input_dir: Path, output_dir, message_format: list[EdifactFormat]) -> No
     for m_format, file in dict_files.items():
         raw_lines = parse_raw_nachrichtenstrukturzeile(file)
         nachrichtenstrukturtabelle = NachrichtenstrukturTabelle.init_raw_table(raw_lines)
+        nachrichtenstrukturtabelle.output_as_csv(m_format, output_dir)
```

Comment:
Nur eine Frage des Geschmacks. 
In dem dataframes von Pandas gibt es die Methode to_csv 
Finde das auch sprechend wenn man schreibt nachrichtenstrukturtabelle.to_csv()

Ist aber nur Frage des Geschmackes